android/lib/src/main/java/org/vosk/android/SpeechService.java

```diff
@@ -314,15 +330,21 @@ class TranscriptionThread extends Thread {
 
         RecognitionListener recognitionListener;
 
+        private boolean isToResetRecognizer = false;
+
         TranscriptionThread(int queueSize, RecognitionListener recognitionListener) {
             this.recordingChunksQueue = new ArrayBlockingQueue<>(queueSize);
             this.recognitionListener = recognitionListener;
         }
 
+        private void isToResetRecognizer() {
```

Comment:
this naming is not very clear/correct:

suggest renaming the method to resetRecognizer(). isToResetRecognizer() looks like a boolean method that returns whether we should reset the recognizer